chain/rosetta-rpc/src/adapters/mod.rs

```diff
@@ -488,6 +488,11 @@ impl From<NearActions> for Vec<crate::models::Operation> {
                 near_primitives::transaction::Action::DeterministicStateInit(_) => {
                     // TODO(#14073): Implement rosetta adapter, probably first requires global contracts, too
                 }
+                near_primitives::action::Action::AddGasKey(_)
+                | near_primitives::action::Action::DeleteGasKey(_)
+                | near_primitives::action::Action::TransferToGasKey(_) => {
+                    // TODO(spice): Implement rosetta adapter, ignored for now.
```

Comment:
please use dedicated label instead of using `spice`, gas keys is an independent project and we will release it separately from spice. we have so many `TODO(spice)` at this point, so it will be easy to miss something gas keys related. I suggest something like `TODO(gas-keys)`